commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -229,3 +235,84 @@ def run(self, args, unknown_args):
         args.playbook = 'deploy_stack.yml'
         unknown_args += ('--tags=users',)
         AnsiblePlaybook(self.parser).run(args, unknown_args)
+
+
+class ManageServices(AnsibleOptions, _AnsiblePlaybookAlias):
```

Comment:
I generally try to name the class the same thing as the command, so by that convention this ought to be called just `Service`.